### DISCUSSION TITLE
Delete Object Browser filters

### DISCUSSION THREAD

**Discussion by ibmiiste** [codefori/vscode-ibmi#1796] [Q&A] [answered]
Hi, 

How can we delete several filters in the Object Browser?

Regards, 

Olivier.

**Comment by sebjulliand** [comment, marked as answer]
Hi @ibmiiste ,
for now you can't select several filters at once to delete them as multi-selection is not enabled in the Object Browser.
The only way to delete those in batch for now is to open your user settings and remove them from there.
![image](https://github.com/codefori/vscode-ibmi/assets/11096890/fee752ad-55ce-4f0d-a373-c2015a54bee1)
![image](https://github.com/codefori/vscode-ibmi/assets/11096890/605fac76-3eb3-4e39-9f1f-aee278957982)

**Reply by ibmiiste** [reply]
Thank you, that will be fine.
